www/docs/admin/banner.php

```diff
@@ -2,19 +2,30 @@
 
 include_once '../../includes/easyparliament/init.php';
 
-$this_page = 'admin_banner';
 
 $db = new ParlDB;
-$banner = new MySociety\TheyWorkForYou\Model\Banner;
+$banner = new MySociety\TheyWorkForYou\Model\AnnoucementManagement;
```

Comment:
Announcement is spelt wrong in the filename/package name (and some of the commit messages).

Reply:
In a lot of places actually.